change/@ni-nimble-angular-471f2c22-eef2-4640-9335-5fc12cb49c02.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Great job churning through this so quick. Wish there wasn't so much boilerplate needed in the wrappers 😫 but looks good 🚢